salt/_modules/metalk8s.py

```diff
@@ -615,3 +622,36 @@ def get_from_map(value, saltenv=None):
         input_data=tmplstr,
         saltenv=saltenv
     )
+
+
+def configure_archive(archive, removed=False):
+    """Add (or remove) a MetalK8s archive in the bootstrap config file."""
+    # raise if the archive does not exist or is invalid
+    archive_info_from_product_txt(archive)
+
+    try:
+        with open(BOOTSTRAP_CONFIG, 'r') as fd:
+            config = yaml.safe_load(fd)
+    except IOError as exc:
+        msg = 'Failed to load bootstrap config file at "{}": {}'.format(
+            BOOTSTRAP_CONFIG, str(exc)
+        )
+        raise CommandExecutionError(message=msg)
+
+    if removed:
+        try:
+            config['archives'].remove(archive)
+        except ValueError:
+            pass
+    else:
+        if archive not in config['archives']:
+            config['archives'].append(archive)
```

Comment:
Minor: To me it could be cool to have a simple message output like "<archive> added" "<archive> removed" "<archive> already present" "<archive> already absent" something like that